.storybook/main.js

```diff
@@ -22,7 +22,6 @@ module.exports = {
             url: false,
           },
         },
-        'resolve-url-loader',
```

Comment:
this was trying to look in design-system/fonts for fonts instead of in the storybook public folder.

Reply:
Interesting :thinking: I thought we needed this for some reason. Upon reading the docs though, it appears to only be used to ensure relative imports in Sass files still work correctly. I don't think we do that anywhere - at least not intentionally.

On a separate note, while looking into this I did find what appears to be a broken import: https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/components/app/account-menu/index.scss#L206

While it might make sense to remove this loader if we're going to stop using relative imports in Sass altogether, I'd recommend not relying upon this for the design system fonts. It's not intuitive where they're being loaded relative to, because of how the bundling works. Also they would all break if we decided to move the final location of the `.css` bundle in the build (e.g. if we moved it into a `.css` directory). Whereas an absolute import is a bit easier to understand maybe.

Reply:
Good call. The font-awesome font urls were being set with a variable that also didn't have a beginning slash. I updated all of these to be relative and the storybook build started showing these fonts again. Huzzah. Restored the resolve-url-loader which can be removed in the future if it isn't necessary.